src/command/controller/data-stream.controller.ts

```diff
@@ -1,30 +1,34 @@
 import { v4 } from 'uuid';
 import { Request } from 'express';
 import { CommandBus } from '@nestjs/cqrs';
+import { UserRole } from '../../user/model/user.model';
+import { Roles } from '../../core/guards/roles.decorator';
+import { RolesGuard } from '../../core/guards/roles.guard';
 import { SensorIdParams } from './model/sensor/sensorid.params';
 import { AddDatastreamBody } from './model/data-stream/add-datastream.body';
 import { AccessJwtAuthGuard } from '../../auth/guard/access-jwt-auth.guard';
 import { DataStreamIdParams } from './model/data-stream/datastreamid.params';
+import { ObservationGoalBody } from './model/data-stream/observation-goal.body';
 import { UpdateDatastreamBody } from './model/data-stream/update-datastream.body';
 import { DomainExceptionFilter } from '../../core/errors/domain-exception.filter';
 import { ApiTags, ApiResponse, ApiOperation, ApiBearerAuth } from '@nestjs/swagger';
 import { AddDataStreamCommand } from '../command/data-stream/add-data-stream.command';
 import { RemoveDataStreamCommand } from '../command/data-stream/remove-data-stream.command';
 import { UpdateDataStreamCommand } from '../command/data-stream/update-data-stream.command';
-import { Controller, Param, Post, Put, Body, Delete, UseFilters, Req, UseGuards } from '@nestjs/common';
-import { ObservationGoalBody } from './model/data-stream/observation-goal.body';
 import { LinkObservationGoalCommand } from '../command/data-stream/link-observationgoal.command';
 import { UnlinkObservationGoalCommand } from '../command/data-stream/unlink-observationgoal.command';
+import { Controller, Param, Post, Put, Body, Delete, UseFilters, Req, UseGuards } from '@nestjs/common';
 
 @ApiBearerAuth()
-@UseGuards(AccessJwtAuthGuard)
-@ApiTags('Data Stream')
+@ApiTags('Datastream')
 @Controller('device')
+@UseGuards(AccessJwtAuthGuard, RolesGuard)
 export class DataStreamController {
     constructor(
         private readonly commandBus: CommandBus,
     ) {}
 
+    @Roles(UserRole.ADMIN)
```

Comment:
Euh ... @kad-floriw , waarom mag alleen een `ADMIN` dit doen?

Reply:
Leek mij goed als iemand die een organisatie joint niet alles kan gaan editen / verwijderen. Die heeft nu alleen view-rechten, tot een organisatie-admin m admin maakt. Maar kunnen ook doen dat je alleen voor delete-operaties admin rechten nodig hebt. Dat is wel een goed idee, dat maakt het iets gebruikersvriendelijker in het begin.

Reply:
Misschien kun je niet zomaar USER worden van een Org. Zou dat niet beter zijn? Dat gebeurt alleen door de ADMIN van een Org. Misschien dat ook alleen een ADMIN de Org details kan aanpassen?

Reply:
Lijkt mij een goed idee, maar voor later ;)